unity/Assets/0Refactored/scenes/experiments/HuygensPrinciple/scripts/InterferencePlate.cs

```diff
@@ -18,15 +19,27 @@ private void Start()
 
         private void Update()
         {
-            //ToDo: Add OnMovement Event to the SlitPlate script
+        }
+
+        protected override void HandleUpdate()
+        {
+        }
+
+        protected override void HandleFixedUpdate()
```

Comment:
I think we can handle this in the HandleUpdate() method.
FixedUpdate is used for being in-step with the physics engine, so anything that needs to be applied to a rigidbody should happen in FixedUpdate. Update, on the other hand, works independently of the physics engine.